src/app/features/ledger/utils/generic-ledger-utils.ts

```diff
@@ -18,3 +28,24 @@ export type SemVerObject = Record<'major' | 'minor' | 'patch', number>;
 export function versionObjectToVersionString(version: SemVerObject) {
   return [version.major, version.minor, version.patch].join('.');
 }
+
+export interface PrepareLedgerDeviceConnectionArgs {
```

Comment:
The naming is wrong here to match the fn name, if that is desired?

Reply:
It's named for the resulting function returned. Any suggestions how to make this clearer?